Cplusplus

dusankojcic opened the pull request:
Sve smo sjebali pomozi nam :dagger:  :D

patrikzvezda committed:
Dodatak 2

dusankojcic committed:
Derivirani Glavonja Sreckovic

dusankojcic committed:
Derivirani Glavonja Sreckovic

aleksamarkoni commented on Josiph/makefile:
```diff
@@ -0,0 +1,11 @@
+	$(CC) -c Josiph.cpp Josiphklasa.h Vojnik.h VojnikGS.h
+
+clean:
+	rm vojnik Josiph.o
```
Mislim da ovo nije najbolje, pogledati tamo kako smo zajedno radili.

aleksamarkoni commented on Josiph/Josiphklasa.h:
```diff
@@ -0,0 +1,24 @@
+  void removeDeadSoldiers();
+  void removeDeadSoldersInPlace();
+  void removeDeadSoldersInPlace2();
+	void ispis();
```
ne koristite ispis nego operator<<

aleksamarkoni commented on Josiph/Josiphklasa.h:
```diff
@@ -0,0 +1,24 @@
+  friend ostream &operator<<( ostream &output, const Josiph &a );
+  Josiph();
+  Josiph(int koliko, int gp);
+  void killThemAll(int m);
```
ovih par metoda mora biti private

aleksamarkoni commented on Josiph/Josiph.cpp:
```diff
@@ -0,0 +1,145 @@
+
+Vojnik::Vojnik() {
+	ziv = true;
+}
```
ovo prebaciti u Vojnik.cpp

aleksamarkoni commented on Josiph/Josiph.cpp:
```diff
@@ -0,0 +1,145 @@
+VojnikGS::Glavonja() {
+	ziv = true;
+	brojZivota = 3;
+}
```
ovo ide u VojnikGS.cpp

aleksamarkoni commented on Josiph/Josiph.cpp:
```diff
@@ -0,0 +1,145 @@
+ostream &operator<<( ostream &output, const Vojnik &a ) {
+	output << "(" << a.poz << "," << (a.ziv?"Ziv":"Mrtav") << ")";
+return output;
+}
```
Ovo gore ide u Vojnik.cpp

pesicnikola023 committed:
work in progress

Zolak94 committed:
fail part 2

aleksamarkoni commented on Josiph/Josiph.cpp:
```diff
@@ -0,0 +1,134 @@
+  }
+}
+
+Vojnik Josiph::dodajVojnike() {
```
promeniti u dodajVojnika

patrikzvezda committed:
Nesto3 sve obrisano.
Merge branch 'Cplusplus' of github.com:aleksamarkoni/paythonovci into Cplusplus

patrikzvezda committed:
Promene

aleksamarkoni commented on Josiph/Josiph.cpp:
```diff
@@ -0,0 +1,136 @@
+	u = koliko;
+	z = koliko;
+	//int poz;
+	krug = new Vojnik[u];
```
ovde staviti srand(time(NULL))

Zolak94 committed:
Lokali

Zolak94 committed:
Namestio sam nasledjivanje klase Kafic

Zolak94 committed:
Ubacen string akcija za klasu kafic

Zolak94 committed:
Dodati pubovi

Zolak94 committed:
dodata funkcija danasnjaAkcija

nikolakoja93 committed:
Domaci pocetni deo

dusankojcic committed:
Treba pomoc

dusankojcic committed:
Merge branch 'Cplusplus' of https://github.com/aleksamarkoni/paythonovci into Cplusplus

nikolakoja93 committed:
Merge branch 'Cplusplus' of github.com:aleksamarkoni/paythonovci into Cplusplus

pesicnikola023 committed:
Nedovrseni zadatak

pesicnikola023 committed:
Merge branch 'Cplusplus' of github.com:aleksamarkoni/paythonovci into Cplusplus

patrikzvezda committed:
Stage one

pesicnikola023 committed:
prebaceni fajlovi u novi folder

Zolak94 committed:
domaci zavrsen

Zolak94 committed:
uredio da bude kao i kod ostalih

dusankojcic committed:
Domaci upis/ispis radi univerzalno

dusankojcic committed:
Dodao main menu

patrikzvezda committed:
Sredjivanje foldera za domaci

dusankojcic committed:
Sve radi prema instrukcijama

dusankojcic committed:
Merge branch 'Cplusplus' of https://github.com/aleksamarkoni/paythonovci into Cplusplus

dusankojcic committed:
Sve radi po instrukcijama

patrikzvezda committed:
Popravka

patrikzvezda committed:
Merge branch 'Cplusplus' of https://github.com/aleksamarkoni/paythonovci into Cplusplus

dusankojcic committed:
Ocistio main.cpp

dusankojcic committed:
Merge branch 'Cplusplus' of https://github.com/aleksamarkoni/paythonovci into Cplusplus

dusankojcic committed:
Ocistio main.cpp

Zolak94 committed:
rename imena fajlova

aleksamarkoni committed:
Neke ispravk

aleksamarkoni committed:
Merge branch 'Cplusplus' of github.com:aleksamarkoni/paythonovci into Cplusplus

aleksamarkoni committed:
Nesto novo opet

aleksamarkoni committed:
Opet nesto novo

aleksamarkoni committed:
IO overriding

dusankojcic committed:
Radi ispis stolova sa random brojem gostiju

dusankojcic committed:
Radi generisanje konobara i baksisa

Zolak94 committed:
zanimljiv problem

Zolak94 committed:
Merge branch 'Cplusplus' of https://github.com/aleksamarkoni/paythonovci into Cplusplus

dusankojcic committed:
matrica radi popunjavanje i ispis

Zolak94 committed:
sve je sredjeno

Zolak94 committed:
Merge branch 'Cplusplus' of https://github.com/aleksamarkoni/paythonovci into Cplusplus

aleksamarkoni committed:
Promene na matrici

aleksamarkoni committed:
Merge branch 'Cplusplus' of github.com:aleksamarkoni/paythonovci into Cplusplus

patrikzvezda committed:
DomaciZadatakMatrica

dusankojcic committed:
Sve osim TODO

dusankojcic committed:
Uradjen TODO dodao assert

aleksamarkoni committed:
matrica implementation

aleksamarkoni committed:
Merge branch 'Cplusplus' of github.com:aleksamarkoni/paythonovci into Cplusplus

Conflicts:
	domaciDusan/domaciMatrica/main.cpp
	domaciDusan/domaciMatrica/matrica.cpp
	domaciDusan/domaciMatrica/matrica.h
	domaciDusan/domaciMatrica/matrica.h.gch
